README.md

```diff
@@ -11,6 +11,13 @@ Stable versions are available on the [Hypothesis plugin page on WordPress.org](h
 5. Click Activate
 6. Visit your WordPress Settings > Hypothesis page to configure how it works on your site
 
+## Development
+
+1. Install `php` and the `dom` and `mbstring` extensions.
+2. [Download composer](https://getcomposer.org/download/), the PHP package manager.
```

Comment:
```suggestion
2. [Download Composer](https://getcomposer.org/download/), the PHP package manager.
```

I'm following the capitalization that [Composer uses in their Getting Started guide](https://getcomposer.org/doc/00-intro.md).